src/sentry/api/endpoints/system_options.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+import sentry
+from sentry import options
+from sentry.api.base import Endpoint
+from sentry.api.permissions import SuperuserPermission
+
+
+class SystemOptionsEndpoint(Endpoint):
+    permission_classes = (SuperuserPermission,)
+
+    def get(self, request):
+        results = {
```

Comment:
not a blocker but we should get some tests in for this

i'd just copy paste some existing, nothing complex, just to ensure things run/work on a basic level